projects/hslayers/src/components/styles/symbolizers/fill-symbolizer/fill-symbolizer.component.html

```diff
@@ -21,23 +24,32 @@
     <span class="badge bg-warning mt-2"
       [title]="'STYLER.warningAttribStoredNotVisualized' | translateHs : {app} ">!</span>
     <div class="p-1">
-      <div ngbDropdown #graphicFillMenu="ngbDropdown" class="d-inline-block" placement="top-right">
-        <button class="btn btn-outline-primary btn-sm" id="dropdownAddFillSymbolizer" ngbDropdownToggle>
-          <span class="icon-plus"></span>
-        </button>
-        <div ngbDropdownMenu aria-labelledby="dropdownAddFillSymbolizer">
-          <button ngbDropdownItem (click)="addSymbolizer('graphicFill', 'Mark')">Mark</button>
-          <button ngbDropdownItem (click)="addSymbolizer('graphicFill', 'Icon')">Icon</button>
+      <div class="btn-group me-3">
```

Comment:
See #3648 is it really the cause for `me-3`?

Reply:
> See #3648 is it really the cause for `me-3`?

No no, that line was only tp get the two buttons together. me-3 might even not be necessary, I just copied it from somewhere